### ISSUE TITLE
Add transformation for present_data

### ISSUE THREAD

**Issue by zacharyoxb** [zacharyoxb/ShapePackingPPO#3]
Use logits to decide on a present/orientation. Partially fill action tensordict for position selection.
